src/js/actions/layers.js

```diff
@@ -740,13 +740,19 @@ define(function (require, exports) {
                 if (firstUninitializedLayers.size === 50) {
                     this.whenIdle(initializeLayersBackground, document);
                 }
+            })
+            .error(function (err) {
+                /* jshint multistr: true */
+                log.error("InitializeLayersBackground action failed for document %s, \
```

Comment:
Nit: I think it would be better to just concatenate two shorter strings here instead of including a JSHint directive, especially since we'd like to get rid of JSHint anyway... 

Also, we use `.catch` elsewhere. Is `.error` a synonym? If so, let's stick with `.catch` for consistency.